components/fonts/tests/font_context.rs

```diff
@@ -206,6 +206,7 @@ mod font_context {
             font_language_override: FontLanguageOverride::normal(),
             line_height: LineHeight::Normal,
             _x_lang: XLang::get_initial_value(),
+            ..FontStyleStruct::initial_values()
```

Comment:
Are the values listed above already the initial values of the `FontStyleStruct`? If so maybe we can just use that instead of manually defining them?

Reply:
Yes, turns out they were all the initial value.